src/utils/links.ts

```diff
@@ -20,20 +20,37 @@ export interface TraceqlFilter {
   value?: (string | string[]);
 }
 
-const title = 'Open in Explore Traces';
-const description = 'Open current query in the Explore Traces app';
+type PluginExtensionExploreContext = {
+  targets: DataQuery[];
+  timeRange: RawTimeRange;
+};
 
 export const linkConfigs: Array<{ targets: string | string[] } & PluginExtensionAddedLinkConfig<PluginExtensionPanelContext>> = [
   {
     targets: PluginExtensionPoints.DashboardPanelMenu,
-    title,
-    description,
+    title: 'Open in Explore Traces',
+    description: 'Open current query in the Explore Traces app',
     path: createAppUrl(),
-    configure: contextToLink,
+    configure: (context?: PluginExtensionPanelContext) => contextToLink(context),
   },
+  {
+    targets: PluginExtensionPoints.ExploreToolbarAction,
+    title: "Open in Grafana Traces Drilldown",
+    description: 'Try our new queryless experience for traces',
+    path: createAppUrl(),
+    configure: (context?: PluginExtensionExploreContext) => contextToLink(context),
+  }
 ];
 
-export function contextToLink<T extends PluginExtensionPanelContext>(context?: T) {
+/**
+ * Props shared between ...
+ */
```

Comment:
Do we want to update this?

Reply:
Thanks 🤦 I've made it more explicit in 85ba35f1f3baeef5577bc2487ace4b901fe71bb8